LICENSE.md

```diff
@@ -1,6 +1,6 @@
 # The MIT License (MIT)
 
-Copyright © 2021 Biometria Ekonomiska Förening
+Copyright © 2022 Biometria Ekonomiska Förening
```

Comment:
I don't think the year needs to/should be updated ?
[relevant link](https://stackoverflow.com/a/2391555/84666)